src/internal/tester/resource_owners.go

```diff
@@ -20,7 +20,7 @@ func M365TenantID(t *testing.T) string {
 	cfg, err := readTestConfig()
 	require.NoError(t, err, "retrieving m365 tenant ID from test configuration", clues.ToCore(err))
 
-	return cfg[TestCfgAzureTenantID]
+	return strings.ToLower(cfg[TestCfgAzureTenantID])
```

Comment:
Are we sure we wanna lowercase IDs? 🤔

Reply:
Do we have any reason not to?

Reply:
I was just wondering if we should lowercase in the first place and if anything would break just because of that. Any specific reason for lowercasing them?

Reply:
Since this is only in tests, let me approve it to unblock.